packages/alfa-rules/src/sia-r87/rule.ts

```diff
@@ -14,10 +15,11 @@ import { hasRole } from "../common/predicate/has-role";
 import { isDocumentElement } from "../common/predicate/is-document-element";
 import { isTabbable } from "../common/predicate/is-tabbable";
 import { isIgnored } from "../common/predicate/is-ignored";
-import { isKeyboardActionable } from "../common/predicate/is-keyboard-actionable";
+import { isVisible } from "../common/predicate/is-visible";
 
 import { Question } from "../common/question";
 import { isAtTheStart } from "../common/predicate/is-at-the-start";
+import { Option } from "@siteimprove/alfa-option";
```

Comment:
This should be moved to the top with the rest of the `@siteimprove/alfa-*` imports.